packages/aws-cdk/test/integ/cli/app/app.js

```diff
@@ -236,6 +249,8 @@ new YourStack(app, `${stackPrefix}-test-2`);
 // Deploy wildcard with parameters does ${stackPrefix}-param-test-*
 new ParameterStack(app, `${stackPrefix}-param-test-1`);
 new OtherParameterStack(app, `${stackPrefix}-param-test-2`);
+// Deploy stack with multiple parameters
+new MultiParameterStack(app, `${stackPrefix}-param-test-3`);
```

Comment:
one last tiny thing - can you also add these param stacks (I just realized the other 2 are missing as well) into the [`cleanup`](https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/test/integ/cli/common.bash#L91-L98) function:

Reply:
donerino